testing/jormungandr-testing-utils/src/testing/fragments/verifier.rs

```diff
@@ -248,3 +248,17 @@ impl FragmentVerifier {
         })
     }
 }
+
+#[derive(Clone, Copy)]
+pub enum ExitStrategy {
```

Comment:
The name is a bit too abstract for strategy of waiting on pending fragments.
Maybe explicitly namespacing it under `fragments::ExitStrategy`, rather than exporting at top level, would be a better style.